src/dropdown-menu/dropdown-menu.component.jsx

```diff
@@ -85,21 +83,18 @@ export default class DropdownMenu extends React.Component {
     });
 
   render() {
-    const { id, menuItems, caret, pullLeft, ...otherProps } = this.props;
+    const { menuItems, caret, pullLeft, ...otherProps } = this.props;
     return (
       <div className="oc-dropdown-menu">
-        <DropdownButton
-          id={id}
-          bsStyle="info"
-          bsSize="xs"
+        <DropdownContainer
```

Comment:
I added max-height and overflow-y to dropdown-multi-select component. Do we need it also for dropdown-menu?